openmicroscopy/bioformats
# ConversionTest: migration to Formats BSD

See https://trello.com/c/jGrPs6c9/16-fix-integration-tests for the general work on the integration/performance test on this branch

As explained in the [developer documentation](https://www.openmicroscopy.org/site/support/bio-formats5.2/developers/components.html), the `test-suite` component is dedicated to Bio-Formats integration tests requiring concrete data.

While reviewing the Ant targets of this component, two of them do not fulfill this requirement: `test-minimal-convert` and `test-tiff-compress`. In addition, these targets also need to be invoked independently (see the BIOFORMATS-DEV-merge-build) and are not covered by the Travis build.

This PR proposes to address this issue for `ConversionTest` by migrating it under the `formats-bsd` unit tests (as all writers are under this component). The test is also registered in the `testng.xml` so that is can be picked up by the default Maven and Ant `test` phases of the component. The `test-minimal-convert` target is removed and the CI jobs will require a corresponding update.

To test this PR:
- check Travis and the CI jobs for the `dev_5_3` branch are green
- check Maven ant/or Ant `test` include the `ConversionTest` as part of the executed test suite
